dipy/workflows/denoise.py

```diff
@@ -337,7 +337,7 @@ def run(self, input_files, slice_axis=2, n_points=3, num_processes=1,
             use as many cores as possible). 0 raises an error.
         out_dir : string, optional
             Output directory. (default current directory)
-        out_unrig : string, optional
+        out_unring : string, optional
```

Comment:
there is a typo on the filename also. Can you update this too? thanks

Reply:
Done in c4d10ed. Push-forced.